enterprise/lib/chat_gpt.rb

```diff
@@ -8,9 +8,9 @@ def initialize(context_sections = '')
     @messages = [system_message(context_sections)]
   end
 
-  def generate_response(input, previous_messages = [], role = 'user')
-    @messages += previous_messages
-    @messages << { 'role': role, 'content': input } if input.present?
+  def generate_response(additional_message: nil, message_history: [], role: 'user')
```

Comment:
This file is not used in Captain. This change would most likely break the rephrase/summarize actions in the editor